src/wp-includes/option.php

```diff
@@ -696,7 +692,7 @@ function add_option( $option, $value = '', $deprecated = '', $autoload = 'yes' )
 	}
 
 	if ( ! wp_installing() ) {
-		if ( 'yes' === $autoload ) {
+		if ( 'yes' === $autoload || 'default' === $autoload ) {
```

Comment:
I wonder whether we should rather use something like `default-yes` here? (And equivalently, there would be a `default-no` value.)

Having only `default` limits us to know whether a value with `no` in the database was explicitly set to `no` or whether it was just using the default and WP core logic decided to set it to `no` (e.g. because it's a too large option value).

Reply:
Can you explain a time we might use `default-no`? I am not sure why this is needed....

Reply:
@spacedmonkey When the option is too large, core may set it to `no`. That would be different from a developer explicitly setting it to `no`, so it would be good to be able to differentiate by it.